### PR TITLE
Fix/working mod

### PR DESCRIPTION
**Pull request by builderjer**
I think this works.  It does for me

### CODE REVIEW

File: neon_tts_plugin_larynx_server/__init__.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 import requests
 from ovos_plugin_manager.templates.tts import TTS, TTSValidator
-from neon_utils.parse_utils import format_speak_tags
+# from neon_utils.parse_utils import format_speak_tags
```
Comment (NeonDaniel) [inline neon_tts_plugin_larynx_server/__init__.py]: @JarbasAI I think this was ported to ovos-plugin-manager, can you confirm which version got that change?

File: neon_tts_plugin_larynx_server/__init__.py [hunk 2]
```diff
@@ -75,16 +75,19 @@ def __init__(self, lang="en-us", config=None):
...
             lang, config, LarynxServerTTSPluginValidator(self), 'wav')
         self.url = config.get("host", "http://tts.neon.ai")
         self.vocoder = config.get("vocoder", "hifi_gan/vctk_small")
         self.noise = config.get("noise", 0.333)
         self.length = config.get("length", 1.0)
         self.denoiser = config.get("denoiser", 0.002)
-        self.voice = config.get("voice", 'mary_ann')
-        if self.voice in self.voice2id:
-            self.voice = self.voice2id[self.voice]
+
```
Comment (NeonDaniel) [inline neon_tts_plugin_larynx_server/__init__.py:87]: Please remove the 2 extra newlines here

File: requirements/requirements.txt [hunk 3]
```diff
@@ -1,4 +1,3 @@
-ovos-plugin-manager~=0.0.4
-SpeechRecognition~=3.8
-neon_utils~=0.6
-requests~=2.26
\ No newline at end of file
+ovos-plugin-manager
```
Comment (NeonDaniel) [inline requirements/requirements.txt]: Are there version conflicts with something here? Unpinned dependencies open up the possibilities for upgrading to an incompatible version of a package and long-running dependency resolution that could download every version of each of these packages to try and find a comparible one

File: setup.py [hunk 4]
```diff
@@ -41,9 +41,9 @@ def get_requirements(requirements_filename: str):
     description='Larynx tts plugin for OVOS / Neon / Mycroft',
     long_description=long_description,
     long_description_content_type='text/markdown',
-    url='https://github.com/NeonGeckoCom/neon-tts-plugin-larynx-server',
-    author='Neongecko',
-    author_email='developers@neon.ai',
+    url='https://github.com/builderjer/neon-tts-plugin-larynx-server',
```
Comment (NeonDaniel) [inline setup.py]: Please revert these changes to setup.py

**Review by NeonDaniel — CHANGES_REQUESTED** [review]

File: requirements/requirements.txt (hunk 3 above)
Comment (builderjer) [reply requirements/requirements.txt]: The change was made in [V0.0.11a4](https://github.com/OpenVoiceOS/OVOS-plugin-manager/tree/V0.0.11a4).  I think I took care of it.

**Comment by NeonJarbas** [comment]
i think this is now obsolete since #15
